JS-Algorithms/challenges.js

```diff
@@ -41,7 +49,14 @@ Invoking "circularArray(2)" should return "["Island", "Japan", "Israel", "German
 const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
-  // YOUR CODE HERE...
+  if (index > 0) {
+    const newIndex = index % COUNTRY_NAMES.length;
+    const ARRAY_AUX1 = COUNTRY_NAMES.slice(newIndex);
```

Comment:
please stick to camelCase naming convention

Reply:
cool, i got it